livestream/control.go

```diff
@@ -48,6 +49,9 @@ func (ls *Livestreamer) Start(ctx context.Context, strm Livestream) error {
 	if err != nil {
 		return fmt.Errorf("failed to update status: %w", err)
 	}
+	if err := ls.CreateEvent(ctx, strm.ID, EventStarted, EventStartedPayload{}); err != nil {
+		log.Printf("failed to log stream start event: %v", err)
```

Comment:
I know it's not mission critical if this fail, but shouldn't the error be handled or passed up?

Reply:
I went back and forth on this - my thinking was that, if this error was passed up, the user would receive a 500 and think the stream hadn't started when it actually had. OTOH if we can update the stream but can't insert an event, something's gone badly wrong anyway 😆 so happy to reconsider.

Reply:
If showtime is having some pickles, informing the user asap would be ideal, at the same time however, this is effectively a logging library and we don't do `err := log.Printf()` 🤷 

I think leaving the `log.Printf` there for the timebeing, and making a new Jira ticket introducing a better logging library so we can improve using the logger ad hoc?

Reply:
Yeah that makes sense. I think for this specific case it makes sense to log-and-continue, because creating the event isn't the _primary_ task of `ls.Start()`.